recipes/raylib/all/conanfile.py

```diff
@@ -72,14 +111,29 @@ def generate(self):
             tc.variables["USE_EXTERNAL_GLFW"] = "ON"
             tc.variables["OPENGL_VERSION"] = "OFF" if not self.options.opengl_version else self.options.opengl_version
         tc.variables["WITH_PIC"] = self.options.get_safe("fPIC", True)
+
+        tc.variables["CUSTOMIZE_BUILD"] = self.options.customize_build
+        if self.options.customize_build:
+            if self._support_custom_modules:
+                tc.variables["SUPPORT_MODULE_RAUDIO"] = self.options.module_raudio
+            tc.variables["SUPPORT_EVENTS_WAITING"] = self.options.events_waiting
+            if self._support_frame_control:
+                tc.variables["SUPPORT_CUSTOM_FRAME_CONTROL"] = self.options.custom_frame_control
+
+            # this makes it include the headers rcamera.h, rgesture.h and rprand.h
+            tc.variables["SUPPORT_CAMERA_SYSTEM"]    = self.options.camera_system
+            tc.variables["SUPPORT_GESTURES_SYSTEM"]  = self.options.gestures_system
+            if self._support_rprand_generator:
+                tc.variables["SUPPORT_RPRAND_GENERATOR"] = self.options.rprand_generator
+
         # Due to a specific logic of cmakedeps_macros.cmake used by CMakeDeps to try to locate shared libs on Windows
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0054"] = "NEW"
+
```

Comment:
```suggestion
        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0091"] = "NEW"
```

https://cmake.org/cmake/help/latest/policy/CMP0091.html
https://c3i.jfrog.io/c3i/misc-v2/logs/pr/24585/14-windows-msvc/raylib/3.5.0//824321e7368fdc0501873cfce5f6e99adc7a95b8-build.txt